src/Sulu/Bundle/SecurityBundle/Build/InitSecurityBuilder.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Build;
+
+use Sulu\Bundle\CoreBundle\Build\SuluBuilder;
+
+/**
+ * Builder for creating anonymous roles.
+ */
+class InitSecurityBuilder extends SuluBuilder
```

Comment:
Shouldn't we call that just `SecurityBuilder`? None of the other builders is prefixed with `Init`